docker-compose.yml

```
@@ -0,0 +1,37 @@
+version: "3.7"
+services:
+  gesso:
+    build: .
+    volumes:
+      - "$PWD:/"
```

Make this match the volume mount configuration in https://github.com/forumone/gesso-wp/pull/240/files
```suggestion
      - ./images:/app/images:cached
      - ./js:/app/js:cached
      - ./source:/app/source:cached
      - type: volume
        source: gesso-pattern-lab
        target: /app/pattern-lab
      - type: volume
        source: gesso-css
        target: /app/css
```